app/src/main/java/org/mozilla/fenix/library/downloads/DownloadFragmentStore.kt

```diff
@@ -16,13 +16,16 @@ import mozilla.components.lib.state.Store
  * @property filePath Full path of the download item
  * @property size The size in bytes of the download item
  * @property contentType The type of file the download is
+ * @property url The full url to the content that should be downloaded
+ * @property status The status that represents every state that a download can be in
  */
 data class DownloadItem(
     val id: String,
     val fileName: String?,
     val filePath: String,
     val size: String,
     val contentType: String?,
+    val url: String,
```

Comment:
I think you intended to make url the second property looking at this now.

Reply:
Addressed